internal/DateTime/tests/DateTimeDecorator-specs.js

```diff
@@ -1,31 +1,34 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import {DateTimeDecorator} from '../';
 
 describe('DateTimeDecorator', () => {
-	test(
-		'should accept an updated JavaScript Date for its value prop',
-		() => {
-			const Picker = DateTimeDecorator({}, function PickerBase () {
-				return <div />;
-			});
-
-			const subject = mount(
-				<Picker
-					title="Date"
-					value={new Date(2000, 0, 1, 12, 30)}
-					locale="en-US"
-				/>
-			);
+	test('should accept an updated JavaScript Date for its value prop', () => {
+		const Picker = DateTimeDecorator({}, function PickerBase ({locale, title, value}) {
+			const minuteValue = value.getMinutes();
+			return <div locale={locale} title={title} >{minuteValue}</div>;
+		});
 
-			subject.setProps({
-				value: new Date(2000, 0, 1, 12, 45)
-			});
+		const {rerender} = render(
+			<Picker
+				title="Date"
+				value={new Date(2000, 0, 1, 12, 30)}
+				locale="en-US"
+			/>
+		);
 
-			const expected = 45;
-			const actual = subject.find('PickerBase').prop('value').getMinutes();
+		rerender(
+			<Picker
+				title="Date"
+				value={new Date(2000, 0, 1, 12, 45)}
+				locale="en-US"
+			/>
+		);
 
-			expect(actual).toBe(expected);
-		}
+		const expected = '45';
+		const actual = screen.getByTitle('Date').textContent;
+		expect(actual).toBe(expected);
+	}
 	);
```

Comment:
Place both these parenthesis on the same line.

Reply:
done